csharp/SSEtoRserve/RServeEvaluator.cs

```diff
@@ -213,7 +213,10 @@ byte[] GetHeader(Metadata Headers, string Key)
                     paramnames += $" {param.Name}";
                 }
                 logger.Info("{0}", paramnames);
-                await AddInputData(scriptHeader.Params.ToArray(), requestStream, rserveConn);
+                if (scriptHeader.Params != null && scriptHeader.Params.Count > 0)
```

Comment:
Is it possible for `Params` to be `null`?

Reply:
Probably not since it will be an empty Collection with count 0 if no params are provided. I guess it was a habit to include a null check.